kittens/choose/main.c

```diff
@@ -128,8 +128,9 @@ run_search(Options *opts, GlobalData *global, const char * const *lines, const s
     Chars chars = {0};
 
     ALLOC_VEC(text_t, chars, 8192 * 20);
+    if (chars.data == NULL) return 1;
     ALLOC_VEC(Candidate, candidates, 8192);
-    if (chars.data == NULL || candidates.data == NULL) return 1;
+    if (candidates.data == NULL) { FREE_VEC(candidates) return 1; }
```

Comment:
Sorry, `FREE_VEC(candidates)` should be `FREE_VEC(chars)`. This was a leftover from some old code of mine where I freed both `candidates` and `chars`, which isn't needed here. I evidently deleted the wrong statement.